services-camel/src/main/java/edu/unc/lib/cdr/ReplicationDestinationUnavailableException.java

```diff
@@ -0,0 +1,10 @@
+package edu.unc.lib.cdr;
+
+@SuppressWarnings("serial")
```

Comment:
It would also make sense for this exception to be a subclass of ReplicationException